three.js/src/threex/arjs-markercontrols.js

```diff
@@ -99,7 +106,7 @@ const MarkerControls = function (context, object3d, parameters) {
 };
 
 MarkerControls.prototype = Object.create(ArBaseControls.prototype);
-MarkerControls.prototype.constructor = MarkerControls;
+//MarkerControls.prototype.constructor = MarkerControls;
```

Comment:
same as in comment https://github.com/AR-js-org/AR.js/pull/506/files#r1071485190